Extensions/Leaderboards/JsExtension.js

```diff
@@ -75,29 +75,57 @@ module.exports = {
       )
       .getCodeExtraInformation()
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
-      .setFunctionName('gdjs.evtTools.leaderboards.setPlayerScore');
+      .setFunctionName('gdjs.evtTools.leaderboards.savePlayerScore');
+
+    extension
+      .addCondition(
```

Comment:
I just had an idea: should we ask, for these 3 conditions related to the score sent to a leaderboard (errored, succeeded, is saving), for the leaderboard id? That's not making things way more complicated for the creator, and this can help the creators to check what happened if they have multiple leaderboards.
I had this idea while reading the code where you mention that a refactoring is needed to store the last player name/last score/last saved at/potentially other stuff indexed by leaderboard id.

I'm not entirely sure but I think this will naturally come as you rework the checks to be per leaderboard rather than global.

Note that the leaderboard display view is single, so it's fine to have the conditions related to this not taking any parameter.